emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java

```diff
@@ -23,6 +19,10 @@ public class ConceptsToEClassesMapping {
   private final Map<Concept, EClass> conceptsToEClasses = new HashMap<>();
   private final Map<Interface, EClass> interfacesToEClasses = new HashMap<>();
 
+  public ConceptsToEClassesMapping() {
```

Comment:
Perhaps later we can add a flag to give the possibility not to register this, in case it is needed

Reply:
The method is public, so interested parties could just call it.